run/markdown-preview/editor/test/runner.sh

```diff
@@ -20,38 +20,40 @@ requireEnv() {
   test "${!1}" || (echo "Environment Variable '$1' not found" && exit 1)
 }
 requireEnv SERVICE_NAME
+requireEnv GOOGLE_CLOUD_PROJECT
 
 # The markdown-preview sample needs to be tested with both the editor and renderer services deployed.
+echo
 echo '---'
-
+echo ":: Deploy the Renderer service"
+echo
 pushd ../renderer
 
 # Version is in the format <PR#>-<GIT COMMIT SHA>.
 # Ensures PR-based triggers of the same branch don't collide if Kokoro attempts
 # to run them concurrently.
-export UPSTREAM_SAMPLE_VERSION="${KOKORO_GIT_COMMIT:-latest}"
-export UPSTREAM_CONTAINER_IMAGE="gcr.io/${GOOGLE_CLOUD_PROJECT}/run-renderer:${UPSTREAM_SAMPLE_VERSION}"
+export UPSTREAM_CONTAINER_IMAGE="gcr.io/${GOOGLE_CLOUD_PROJECT}/run-renderer:${SAMPLE_VERSION:-latest}"
 
 # Build the Renderer service
 set -x
 gcloud builds submit --tag="${UPSTREAM_CONTAINER_IMAGE}"
 set +x
 
-pushd ../editor
```

Comment:
Do we not need to be in the editor folder to access test/deploy.sh?

Reply:
This is calling `run/markdown-preview/renderer/test/deploy.sh`.